### ISSUE TITLE
Allow value binding on Validation.Enabled attribute

### ISSUE THREAD

**Issue by symbiogenesis** [riganti/dotvvm#491]
This would be a nice feature. For now, I'm using a workaround with two versions of a button, one enabled one not, and binding on the visibility property.

**Comment by tomasherceg** [comment]
@exyi I don't think this is a good idea because of the security reasons - it would be quite easy to turn the validation off for some parts of the form.

**Comment by symbiogenesis** [comment]
I generally assume that for security, you shouldn't trust the JSON at all, and thus client-side validation doesn't add much security. Only server-side does.

**Comment by tomasherceg** [comment]
Of course, but if you succeed to change the JSON so the expression used in `Validation.Enabled` evaluates to `false` on the server, you will be able to to turn off even the server-side validation.

**Comment by exyi** [comment]
I don't think it would be a big security issue, as it would like now when the binding is not present. And having two buttons that differ only in `Validation.Enabled` property has the same issue and is clearly pretty ugly. And you can always use the view model protection feature.
